mammoth/docx/body_xml.py

```diff
@@ -350,30 +349,50 @@ def break_(element):
             warning = results.warning("Unsupported break type: {0}".format(break_type))
             return _empty_result_with_message(warning)
 
-    
     def inline(element):
+        content_info = {'width': '', 'height': '', 'alt_text': ''}
+        
         properties = element.find_child_or_null("wp:docPr").attributes
+
         if properties.get("descr", "").strip():
-            alt_text = properties.get("descr")
+            content_info['alt_text'] = properties.get("descr")
         else:
-            alt_text = properties.get("title")
+            content_info['alt_text'] = properties.get("title")
+
+        extAttributes = element.find_child("a:graphic") \
+                        .find_child("a:graphicData") \
+                        .find_child("pic:pic") \
+                        .find_child("pic:spPr") \
+                        .find_child("a:xfrm") \
+                        .find_child("a:ext").attributes
+    
+        content_info['width'] = extAttributes.get("cx")
+        content_info['height'] = extAttributes.get("cy")
```

Comment:
Is this the dimensions of the original image, or of what the user sees? The relevance being that applying the dimensions might distort the image if other attributes, such those that crop the image, aren't being applied.

Reply:
It should be original, but let me double check that! Wonder how we can write this in a test?

Reply:
I'm not sure there's anything worth adding to a test. I'd say it's more a question of checking the spec and actual documents.

Reply:
I thought this was the original image size as all transformations seem to be inside pic:blipFill. But reading more about it. It is in fact the bounding box. :( Will have to do a bit more research here.